instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/jaxrs-client-2.0-jersey-2.0-javaagent.gradle

```diff
@@ -4,7 +4,7 @@ muzzle {
   pass {
     group = "org.glassfish.jersey.core"
     module = "jersey-client"
-    versions = "[2.0,)"
+    versions = "[2.0,3.0.0)"
```

Comment:
Just wondering: why did you add the version limit here?

Reply:
Trying to fix sudden muzzle failure :) It seems that there is a new recent version and muzzle failed on that.